src/model_metric_reporter.cpp

```diff
@@ -323,4 +323,168 @@ ModelMetricReporter::ModelMetricReporter(const MetricConfig* metricConfig, Metri
     }
 }
 
+MediapipeServableMetricReporter::MediapipeServableMetricReporter(const MetricConfig* metricConfig, MetricRegistry* registry, const std::string& graphName) :
+    registry(registry) {
+    if (!registry) {
+        return;
+    }
+
+    if (!metricConfig || !metricConfig->metricsEnabled) {
```

Comment:
Those two can be nullptr if metrics are not enabled?

Reply:
metricConfig can be nullptr only in tests, that was always workaround for setups that do not require metrics testing